mobile/lib/features/wallet/wallet_theme.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+@immutable
```

Comment:
💡 O cool, I did not know one can mark an entire class as immutable.

Reply:
That is copied from TradeTheme :sweat_smile:

Reply:
hahaha, cool - then I did apparently know at some point and forgot it again